### ISSUE TITLE
[1.12.2] menril tree biome config options

### ISSUE THREAD

**Issue by xemnes** [CyclopsMC/IntegratedDynamics#934]
since i use RTG for my world gen, im unable to have the meneglin biome spawn in my world. i also have banded climats from geographicraft enabled so the biomes at the center of the world are warm biomes like jungles, plains and savannas. ive increased the spawn chance for menril to 1 and flew about a bit only to find about 2 menril trees in total in a small forest biome quite a far distance off. not a single menril tree in the plains or jungle biomes. i noticed they spawned more often in cooler biomes but these are 1000s of blocks away from spawn which isnt ideal if you wanted to settle in a warm biome.

is it possible to have an option in the config to choose which biomes the trees spawn in and perhaps configurable weights for each defined biome?
for example:
minecraft:jungle=20
minecraft:plains=140
biomesoplenty:lush_desert=250
biomesoplenty:tropical_rainforest=60

**Comment by rubensworks** [comment]
Thanks for the suggestion!

**Comment by xemnes** [comment]
youre welcome, though how likely is it to be added and how long would it take? its quite an issue for me at the moment as theres no menril to go around.

**Comment by rubensworks** [comment]
I will probably never have time to implement this, so this will only be implemented once someone else comes a long and provides a pull request.

**Comment by xemnes** [comment]
right, ok. since i use dynamic trees, the dev for the menril addon for DT has kindly offered to add in the ability via DT's ability to modify tree worldgen which is a solution. though the feature implemented into the main mod is still a nice idea for the future.
